torch/jit/_recursive.py

```diff
@@ -329,6 +340,10 @@ def init_fn(script_module):
                 cpp_module._register_parameter(name, orig_value, False)
             elif isinstance(orig_value, torch.jit.Attribute):
```

Comment:
This copypasta makes this PR more confusing. There should only be one call to _register_attribute in this loop, and it should be clear that it is the fallthrough pathway. I.e. previously modules were going through _register_attribute and the original value is (presumably) the already scripted module.